src/test/java/net/barrage/school/java/ecatalog/app/HibernateExampleTest.java

```diff
@@ -69,6 +90,7 @@ class Product {
 
     private String imageUrl;
 
+    @ToString.Exclude
```

Comment:
We can't print this field in `toString` cause recursion. product -> merchant -> products[] -> original product.